docker/Dockerfile-build

```diff
@@ -75,4 +75,14 @@ RUN mkdir -p /tmp/setup-jna \
     && cd ../ \
     && rm -rf ./setup-jna
 
+RUN mkdir -p /tmp/setup-kotlinx \
+    && cd /tmp/setup-kotlinx \
+    && curl -o kotlinx-coroutines-core-jvm.jar https://repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-coroutines-core-jvm/1.6.4/kotlinx-coroutines-core-jvm-1.6.4.jar \
+    # XXX TODO: should check a sha256sum or something here...
```

Comment:
These are the checksums that I was thinking about, but I realize this comment was just copied from the swift line above.  This doesn't seem like a regression to me.  Also, this container just handles the CI, so I don't think the risk is large.  Let's not block merging on this one.